metricbeat/module/galera/status/status.go

```diff
@@ -0,0 +1,113 @@
+package status
+
+
+import (
+	"database/sql"
+
+	"github.com/elastic/beats/metricbeat/mb"
+	"github.com/pkg/errors"
+	"github.com/elastic/beats/libbeat/logp"
+	"github.com/elastic/beats/libbeat/common"
+
+	mbSql "github.com/elastic/beats/metricbeat/module/mysql"
+)
+
+
+var (
+	debugf = logp.MakeDebug("galera-status")
+)
+
+
+// init registers the MetricSet with the central registry.
+func init() {
+	if err := mb.Registry.AddMetricSet("galera", "status", New, mbSql.ParseDSN); err != nil {
+		panic(err)
+	}
+}
+
+// MetricSet for fetching Galera-MySQL server status
+type MetricSet struct {
+	mb.BaseMetricSet
+	db *sql.DB
+	queryMode string
+}
+
+// New create a new instance of the MetricSet
+// Loads query_mode config setting from the config file
+func New(base mb.BaseMetricSet) (mb.MetricSet, error) {
```

Comment:
I think we should flag this as experimental for the first version, could you add something similar to this? https://github.com/elastic/beats/blob/746f8b9254f654148b2653e2c473afa1bfb93629/metricbeat/module/ceph/pool_disk/pool_disk.go#L36